sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1674,6 +1886,26 @@ export interface ListBlobsFlatSegmentResponse {
     serviceEndpoint: string;
 }
 
+// @public
+export interface ListBlobsFlatSegmentResponseModel {
+    // (undocumented)
+    containerName: string;
+    // (undocumented)
+    continuationToken?: string;
+    // (undocumented)
+    marker?: string;
+    // (undocumented)
+    maxPageSize?: number;
+    // (undocumented)
+    prefix?: string;
+    // Warning: (ae-forgotten-export) The symbol "BlobFlatListSegment" needs to be exported by the entry point index.d.ts
```

Comment:
seems like there is a naming conflict here as well as a missing export

Reply:
fixed